packages/docs/docs/tutorials/bots/hl7-into-fhir.md

```diff
@@ -0,0 +1,120 @@
+
+# Converting an HL7 Feed into FHIR Objects
+
+HL7 interfaces are common in healthcare, and widely supported by legacy EHRs, RIS/PACS systems, lab machines and more.  Some common HL7 feeds include:
+
+* Admission, Discharge and Transfer fees (ADT Feed)
+* Observation Results (OBX Feed)
+* Scheduling Information Unsolicited (SIU feed)
+
+These feeds contain a wealth of information, but can be hard to manage, programmatically.  Medplum makes consuming and managing these feeds straightforward.
+
+## This guide will show you
+
+* How to integrate Medplum with an HL7 feed bi-directionally, receiving messages and sending back confirmation messages.
+* How to convert those HL7 messages into FHIR objects
+
+## The Workflow
+
+When we this implementation is complete we will:
+
+* Recieve an HL7 message over HTTPS from another EHR that produces HL7 feeds
+* Create FHIR objects that capture the parts of the feed we care about.
+* Respond to the EHR with an HL7 acknowledgment that the message was received
+* (Optional) Send a notification to another application, indicating that new FHIR resources are available.
+
+## The Implementation
+
+### Account and Policy Setup
+
+* Make sure you have an account on Medplum, if not, [register](https://app.medplum.com/register).
+* Create a [ClientApplication](https://app.medplum.com/admin/project) on Medplum called "ADT Bot Client Application".
+* (Optional) Create a very restrictive [AccessPolicy](https://app.medplum.com/AccessPolicy) called "ADT Bot Policy", make it so that the policy only allows readwrite on the Patient object.
+* (Optional) In the [ProjectAdmin dashboard](https://app.medplum.com/admin/project) apply the "ADT Bot Policy" policy to the `ClientApplication` by clicking `Access`.
+
+### Bot Setup
+
+* Linking the ADT feed to Medplum is done through [Medplum Bots](https://app.medplum.com/Bot).
+* At a high level, each Bot exposes an endpoint and HL7 messages are posted over HTTPS to that endpoint.
+* The HL7 message is parsed and then converted to a corresponding FHIR object.  
+* (Optional) If needed, you can link a [Subscription](https://app.medplum.com/Subscription) to the FHIR objects that the Bot creates to notify downstream applications that new data is available.
+
+* Make the Easypost subscription bot that will listen for webhooks
+  * First, [create a bot](https://app.medplum.com/admin/project) called ADT Handler Bot and save it
+  * Paste the code below into the Bot you created and save.
+
+  ```js
+  export async function handler(medplum, event) {
+    const input = event.input;
+    //Log Message Type
```

Comment:
Formatting - space after `//`

Reply:
Oops, my bad.  I thought my prettier would take care of this.  Will clean it up